packages/invoices/src/InboundShipment/ListView/ListView.tsx

```diff
@@ -3,55 +3,38 @@ import {
   useNavigate,
   DataTable,
   useColumns,
-  useListData,
   getNameAndColorColumn,
   TableProvider,
   createTableStore,
   InvoiceNodeStatus,
-  useNotification,
   generateUUID,
   useTranslation,
-  useQueryParams,
 } from '@openmsupply-client/common';
-import { getInboundShipmentListViewApi } from './api';
 import { NameSearchModal } from '@openmsupply-client/system';
 import { Toolbar } from './Toolbar';
 import { AppBarButtons } from './AppBarButtons';
 import { InvoiceRow } from '../../types';
 import { getStatusTranslator } from '../../utils';
-import { useInboundShipmentApi } from '../api';
+import { useInbounds, useCreateInbound } from '../api';
 
 export const InboundListView: FC = () => {
+  const { mutate } = useCreateInbound();
   const navigate = useNavigate();
-  const { error } = useNotification();
-  const api = useInboundShipmentApi();
-  const { storeId } = useQueryParams({ initialSortBy: { key: 'id' } });
   const {
-    totalCount,
     data,
     isLoading,
-    onDelete,
-    onUpdate,
     sortBy,
     onChangeSortBy,
-    onCreate,
     onChangePage,
     pagination,
     filter,
-    invalidate,
-  } = useListData(
-    {
-      initialSortBy: { key: 'otherPartyName' },
-      initialFilterBy: { type: { equalTo: 'INBOUND_SHIPMENT' } },
-    },
-    'invoice',
-    getInboundShipmentListViewApi(api, storeId)
-  );
+  } = useInbounds();
+
   const t = useTranslation();
 
   const columns = useColumns<InvoiceRow>(
     [
-      [getNameAndColorColumn(), { setter: onUpdate }],
+      [getNameAndColorColumn(), { setter: () => {} }],
```

Comment:
Will make an issue for this - similar is done for the requisition list views, so will do them all